### PR TITLE
Add rule for api identifiers.

### PR DESCRIPTION
**Pull request by tkrop**
For API management we want to introduce API identifiers, that allow to track histories of APIs independent of volatile API titles. For more information see the internal proposal of [API identifiers for API histories](https://docs.google.com/document/d/1lRXcTZbZMnFeTVCaazitSWxSdKXGWkOUn99Gr-huNXg/edit#).

### CODE REVIEW

File: chapters/compatibility.adoc [hunk 1]
```diff
@@ -331,3 +331,54 @@ version 1.5.1  -- first draft for next review and API change cycle; compatible w
...
+swagger: 2.0,
+info:
+  x-api-id: d0184f38-b98d-11e7-9c56-68f728c1ba70
+  title: Parcel service API
+  description: API for <...>
+  version: 1.0.0
+  <...>
+----
+
+If no API identifier is provided, a virtual API identifier is assigned to
```
Comment (maxim-tschumak) [inline chapters/compatibility.adoc]: It's an implementation detail of the fallback method to identify APIs. Do we really have to state this in the guideline rule? If yes, we should probably link this paragraph to API Discovery process and/or API Repository in order to provide some background information (where is the API id used?).

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Comment by tkrop** [comment]
👍

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**